tms_objects2.erb

```diff
@@ -248,7 +248,29 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
                     <lido:appelationValue lido:pref="<%= a["NameType"] %>"><%= a["DisplayName"] %></lido:appelationValue>  
             <% end %>  
           <% end %>  
+        <% end %>   <%# ConAltNames |a| %>
+        <% ConAltNames.each do |a| %>  
+        <% if a["DoDn"] == c["DoDn"] and a["NameType"] == "preferred" %>  
+          <% if a["DisplayDate"] == "" %>  
```

Comment:
Judging by the space in the output when checking 61579 **if a["DisplayDate"] == ""** isn't working, might have to be **if a["DisplayDate"].nil?**